consensus/src/experimental/buffer_manager.rs

```diff
@@ -170,10 +184,16 @@ impl BufferManager {
         );
         if self.execution_root.is_some() {
             let ordered_blocks = self.buffer.get(&self.execution_root).get_blocks().clone();
-            self.execution_phase_tx
-                .send(self.create_new_request(ExecutionRequest { ordered_blocks }))
-                .await
-                .expect("Failed to send execution request")
+            let request = self.create_new_request(ExecutionRequest { ordered_blocks });
+            if cursor == self.execution_root {
+                let sender = self.execution_phase_tx.clone();
+                Self::spawn_retry_request(sender, request, Duration::from_millis(100));
```

Comment:
What's the reasoning behind the delay? Can you add a comment to clarify?

Reply:
it's to avoid busy loop, if it fails, it'll enter a loop of retrying